automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/AsyncImportSpec.scala

```diff
@@ -19,20 +24,20 @@ import org.scalatest.time.{Minutes, Seconds, Span}
 import org.scalatest.{FreeSpec, Matchers}
 import spray.json._
 
+import java.util.UUID
 import java.util.concurrent.TimeUnit
+import javax.xml.parsers.FactoryConfigurationError
+import scala.collection.JavaConverters._
 import scala.io.Source
 import scala.util.{Failure, Success, Try}
 
 class AsyncImportSpec extends FreeSpec with Matchers with Eventually with ScalaFutures
   with BillingFixtures with WorkspaceFixtures with Orchestration {
 
-  val owner: Credentials = UserPool.chooseProjectOwner
-  val ownerAuthToken: AuthToken = owner.makeAuthToken()
-
```

Comment:
removed these class-level variables and replaced them with test-specific ones for clarity.